### PR TITLE
Add ability to leave comments in flat files

### PR DESCRIPTION
**Pull request by rdooley**
* just dont upload them
* #9

## Post merge
- [ ] cut a new release

### CODE REVIEW

File: cli/write.go [hunk 1]
```diff
@@ -36,6 +36,10 @@ func Write(table string, path string, ds service.Dynamo, w io.Writer) {
 			// Ignore blank lines
 			continue
 		}
+		if strings.HasPrefix(line, "#") {
```
Comment (verkestk) [inline cli/write.go:39]: Quick question:

Are multi-line string values supported? If so, could a line legitimately start with "#" in this case?

Comment (rdooley) [reply cli/write.go:39]: They are hmmmm.
Let me add a failing test case for that and work on it

Comment (rdooley) [reply cli/write.go:39]: I decided to note that multilines have to be indented (which was already true) and add an example of  `#` starting an indented section of a multi-line string value

**Review by verkestk — APPROVED** [review]

**Comment by rdooley** [comment]
Release coming after i do some work related things

**Pull request merged**